test/testdata/SerialEcho.ino

```diff
@@ -0,0 +1,18 @@
+int incomingByte = 0;    // for incoming serial data
```

Comment:
Put this in `test/testdata/SerialEcho/SerialEcho.ino` to ease compilation and upload for testing.